docs/asciidoc/exportCypher.adoc

```diff
@@ -17,6 +17,17 @@ It is possible to choose between three export formats:
 * `cypher-shell`: for Cypher shell
 * `plain`: doesn't output begin / commit / await just plain Cypher
 
+You can also use the Optimizations like: `useOptimizations: {config}`
+Config could have this params:
+
+* `unwindBatchSize`:  (default 100)
+* `type`: possible values ('NONE', 'UNWIND_BATCH', 'UNWIND_BATCH_PARAMS') (default 'UNWIND_BATCH')
+
+With `NONE` it will export the file with `CREATE` statement;
```

Comment:
with individual statements, which might be MERGE/MATCH or CREATE depending on the strategy.

Reply:
Do we want to handle this in a separate PR or does it already work?